### DISCUSSION TITLE
aarch64 binary

### DISCUSSION THREAD

**Discussion by beremour** [zbm-dev/zfsbootmenu#525] [Questions] [answered]
Is there an arm64 EFI-binary ?

**Comment by classabbyamp** [comment, marked as answer]
Not currently

**Comment by beremour** [comment]
thanx anyway

**Comment by zdykstra** [comment]
To clarify - you can just build your own on your local system. The step-by-step documentation applies more or less as equally to aarch64 as it does to x86_64.
